docs/guide/siteaccess.md

```diff
@@ -193,6 +193,22 @@ ezpublish:
             var_dir: var/site
 ```
 
+Be aware that the `default` scope overrides both back and front views.
+For example, the following configuration overrides both the front template for articles
+and the template used in the Back Office:
```

Comment:
```suggestion
Be aware that the `default` scope is  used by both backend and frontend siteaccesses.
For example, the following configuration sets article template globally, and is used across all siteaccesses unless also set in a higher priority scope:
```


Afaik the config reading order is default, group, siteacess and then global or whatever it is called. If so `default` does not override anything, but it is set as baseline config implicit for all siteaccesses